core/src/main/scala/org/bitcoins/core/protocol/Address.scala

```diff
@@ -79,12 +79,10 @@ sealed abstract class P2SHAddress extends BitcoinAddress {
   */
 sealed abstract class Bech32Address extends BitcoinAddress {
 
-  def hrp: BtcHumanReadablePart
+  val hrp: BtcHumanReadablePart = BtcHumanReadablePart(networkParameters)
```

Comment:
make this lazy just for safety since this is an abstract class, initialization stuff can get weird